tslint.json

```diff
@@ -25,6 +39,18 @@
     "quotemark": [true, "single"],
     "semicolon": true,
     "triple-equals": true,
-    "variable-name": false
+    "variable-name": false,
+    "space-before-function-paren": [true, "never"],
+    "whitespace": [
+      true,
+      "check-branch",
+      "check-decl",
+      "check-module",
+      "check-operator",
+      "check-preblock",
+      "check-separator",
+      "check-type",
+      "check-typecast"
```

Comment:
Absolutely agree, removed.
Only advantage to have it in place is to force consistency. It would be probably better to make a feature request to tslint and have a possibility to decide about space for each option separately.